test/unit/light/unit_LcdLedDriver.cpp

```
@@ -197,3 +197,18 @@ TEST_CASE("LcdLedDriver loopbackRxPin tracks the loopbackTest toggle") {
     }
     CHECK(found);
 }
+
+// loopbackTxPin (optional lane-0 TX override) is bound always, hidden until the
+// test is on — same conditional-control contract as loopbackRxPin.
+TEST_CASE("LcdLedDriver loopbackTxPin tracks the loopbackTest toggle") {
+    mm::LcdLedDriver d;
```

Clang (14.0.6)

[warning] variable name 'd' is too short, expected at least 3 characters

(readability-identifier-length)